readthedocs/proxito/urls.py

```diff
@@ -108,6 +108,19 @@
         name='docs_detail',
     ),
 
+    # Hack /en/latest so it redirects properly
+    # We don't want to serve the docs here,
+    # because it's at a different level of serving so relative links break.
+    url(
+        (
+            r'^(?:projects/(?P<subproject_slug>{project_slug})/)?'
+            r'(?P<lang_slug>{lang_slug})/'
+            r'(?P<version_slug>{version_slug})$'.format(**pattern_opts)
+        ),
+        ServeDocs.as_view(redirect_root=True),
+        name='docs_detail',
```

Comment:
Use a different name here,

```suggestion
        name='docs_detail_directory_indexing',
```